.changeset/@graphql-tools_load-7588-dependencies.md

```
@@ -0,0 +1,6 @@
+---
+"@graphql-tools/load": patch
+---
+dependencies updates:
+  - Updated dependency [`@graphql-tools/schema@^10.0.27` ↗︎](https://www.npmjs.com/package/@graphql-tools/schema/v/10.0.27) (from `^10.0.26`, in `dependencies`)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)